run-local.sh

```diff
@@ -13,6 +13,6 @@ EOC
 if [ \$? -eq 1 ]; then
     pew toggleglobalsitepackages
 fi
-guake --no-startup-script
+PYTHONPATH=. python3 guake/main.py --no-startup-script
```

Comment:
why is this needed?

Reply:
I had guake installed from the repo, so `/usr/lib/python3.6/site-packages/guake/globals.py` existed. Then I did some changes to guake/globals.py for example 
```python
...
import sys
sys.exit(1)
...
```
and ran `make dev ; make run` guake just ran fine without exiting. Turns out that `pew toggleglobalsitepackages` also toggles the guake-site-package and the directory /usr/... has priority over ~/.local/... .
So setting the PYTHONPATH to . will set the priority back to the source directory.

Reply:
great!